PokedexMentoria/Feature/Details/DetailsViews/StatsViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  StatsViewController.swift
+//  PokedexMentoria
+//
+//  Created by Matheus Rodrigues Araujo on 26/04/23.
+//
+
+import UIKit
+
+final class StatsViewController: UIViewController {
+    
+    // MARK: - UI Elements
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .plain)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(ProgressBarCell.self, forCellReuseIdentifier: ProgressBarCell.identifier)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.allowsSelection = false
+        tableView.separatorColor = .clear
+        return tableView
+    }()
+    
+    // MARK: - Variables
+    var stats: [BaseStats] = []
+    private let heightForRow: CGFloat = 55
+    
+    // MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        makeLayout()
+    }
+    
+    func makeLayout() {
+        view.addSubview(tableView)
+        
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 0)
+        ])
+    }
+}
+
+extension StatsViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        stats.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ProgressBarCell.identifier, for: indexPath) as? ProgressBarCell else { return UITableViewCell() }
+        cell.setUpProgressBar(stats: stats[indexPath.row])
```

Comment:
```suggestion
        cell.setUpProgressBar(stats: viewModel.stats[indexPath.row])
```
será que não rolaria disso ficar na viewModel?

Reply:
Essa classe não tem referencia da viewModel.
eu tentei colocar mas ai ia ou ter q passar a viewModel nova via delegate, colocar um guard let ai ou passar por init e o delegate só chamar um reloadData.
Eu achei melhor deixar assim msm, fica mais simples